infra/lambda_functions/private/handledisconnect/app.js

```diff
@@ -0,0 +1,168 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
this function is used whenever we want to remove a user and send a notification to all the others about it. It's also used when we find a stale connection